defaults/main.yml

```diff
@@ -679,6 +730,27 @@ gitlab_pages_dest: '{{ gitlab_src_path + "/" + gitlab_pages_repo.split("://")[1]
 gitlab_pages_checkout: '{{ gitlab_app_root_path + "/gitlab-pages" }}'
                                                                    # ]]]
                                                                    # ]]]
+# Gitaly source code [[[
+# --------------------------------
+
+# .. envvar:: gitaly_repo [[[
```

Comment:
The headers also need to be renamed, otherwise references in the documentation will be broken.